README.md

```diff
@@ -19,7 +19,7 @@ To get a local copy up and running follow these simple example steps. (NPM)
 2. Navigate to the directory where you want your project cloned
 3. Clone the project using `git clone` or just copy the command below:
 ```  
-   git clone https://github.com/didierganthier/getting-started-jest.git
+   git clone https://github.com/didierganthier/leaderboard-project.git
 ```   
 4. Open the project with your IDE. I use [Visual Studio Code](https://code.visualstudio.com/download)
 5. Run `npm install` to get the dependencies
```

Comment:
- [ ] Please add the instructions on how to run webpack build `npm run build` and how to run webpack dev server `npm run start`. To make your readme file more professional and user friendly.